pkg/cmd/cli/cmd/version.go

```diff
@@ -93,7 +93,7 @@ func (o *VersionOptions) Complete(cmd *cobra.Command, f *clientcmd.Factory, out
 
 // RunVersion attempts to display client and server versions for Kubernetes and OpenShift
 func (o VersionOptions) RunVersion() error {
-	fmt.Fprintf(o.Out, "%s %v\n", o.BaseName, version.Get())
```

Comment:
Not the right fix, this breaks versions for all the commands that reuse this code.